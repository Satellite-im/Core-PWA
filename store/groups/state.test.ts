import { expect } from '@jest/globals'
import InitialGroupsState from '~/store/groups/state'

describe('init', () => {
  let inst: any

  beforeEach(() => {
    inst = InitialGroupsState()
  })

  it('should return the initial settings state', () => {
    expect(inst).toMatchSnapshot()
  })

  it('should not return the initial settings state', () => {
    expect(inst).not.toEqual({})
  })
})
