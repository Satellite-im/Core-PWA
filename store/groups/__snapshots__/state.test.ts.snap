// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`init should return the initial settings state 1`] = `
Object {
  "all": Array [],
  "subscriptionId": null,
}
`;
